Sources/OktaDirectAuth/Internal/Extensions/Status+ErrorInitializers.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) 2024-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+
+extension DirectAuthenticationFlow.Status {
+    @usableFromInline
```

Comment:
Why there is so much love to inlinable methods? No concern if sdk is distributed as static lib, but in case of dynamic(let's say linux .so) we are committing ourselves that any change in inlinable function will be a breaking change